authentik/core/models.py

```diff
@@ -167,6 +169,9 @@ class User(SerializerModel, GuardianUserMixin, AbstractUser):
     ak_groups = models.ManyToManyField("Group", related_name="users")
     password_change_date = models.DateTimeField(auto_now_add=True)
 
+    krb5_keys = models.JSONField(blank=True, default=dict)
```

Comment:
I think it would be better to have this as a separate model in the kerberos provider that references a user (and maybe a provider) instead of fields directly on the user

Reply:
Agreed, we'll keep this until a refactoring round where we'll also de-duplicate the keys rotation logic and the keytab generation code

Reply:
Done in https://github.com/goauthentik/authentik/pull/6391/commits/48524cd43373e6f48cb7674ca8392ee95a59334d and https://github.com/goauthentik/authentik/pull/6391/commits/7f233b4ec3e45a952922bea0674a633b12898096
let us know what you think